### DISCUSSION TITLE
What are the user research findings around va.gov form confirmation pages?

### DISCUSSION THREAD

**Discussion by GnatalieH** [department-of-veterans-affairs/vets-design-system-documentation#547] [Announcements]
At the end of a form, the user is presented with an editable view that lets them check their answers before submitting the form. If you have researched forms on va.gov and have findings related to the Review page experience, please comment on this ticket and link to any relevant docs. Thank you in advance!

[Review page documentation on design.va.gov](https://design.va.gov/patterns/form-templates#review-section).

Example:

![Step 5 review - mobile](https://user-images.githubusercontent.com/88347129/134039659-652487cc-dee0-437a-b7c1-78213dc2ccca.png)

**Comment by joshkimux** [comment]
The current review page experience has critical accessibility issues around the length and complexity that comes with being able to edit all answers on the same page. Would we be able to consider using a [check answer pattern similar to gov.uk](https://design-system.service.gov.uk/patterns/check-answers/)? For example:

![image](https://user-images.githubusercontent.com/14154792/134044443-ba3e6a81-0b21-49a9-afe6-7ddf70add647.png)

**Reply by GnatalieH** [reply]
Yes, @joshkimux that's how we want to update the pattern, to be more in alignment with the Check answers pattern. Thanks.

**Comment by GnatalieH** [comment]
From Dené Gabaldón in Slack thread:

As a UX designer, I always expect this page to not test well, BUT in all my research on forms, the vast majority (almost all) of my research participants interact with it just fine. -This is not to say it can’t be improved, and obviously Josh pointed out glaring accessibility issues that need to be addressed! _(NOTE: her research has only been on the 10-10EZ and 10-10CG)_

Reply from Josh Kim:

^Yes, and adding more context to this point-- the usability issues/severity may vary on the length and complexity of a form. For forms that require several days to complete like the 526 one (that can't be tested in a one hour session), I'd imagine it wouldn't be a stretch hypothesis to anticipate a net negative experience.
